Makefile

```diff
@@ -100,11 +100,13 @@ ifdef TEST_WINDOWS
 		--set driver.azureGoSDKLogLevel=INFO \
 		--set controller.replicas=1 \
 		--set controller.logLevel=6 \
-		--set node.logLevel=6
+		--set node.logLevel=6 \
+		--disable-openapi-validation
```

Comment:
there is `WINDOWS_USE_HOST_PROCESS_CONTAINERS` env var in test-infra, we could only `disable-openapi-validation` when `WINDOWS_USE_HOST_PROCESS_CONTAINERS` is set and also run host process test in e2e test:

https://github.com/kubernetes/test-infra/blob/b02ce7741ab34242f95f33fe680d8507c3060f09/config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml#L663-L664